src/components/Toggle/ListToggle.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import { TYPE } from '../../theme'
+
+const Wrapper = styled.button<{ isActive?: boolean; activeElement?: boolean }>`
```

Comment:
How `isActive` and `activeElement` are used here?

Reply:
These props were unused. Removed here 4dfeb5c7584e2f44b37b2a08087102856a031281